.github/workflows/publish_build.yml

```
@@ -0,0 +1,84 @@
+        uses: stefanzweifel/git-auto-commit-action@v4
+        with:
+          commit_message: Declare alpha stable
+          branch: dev
+      - name: Push dev -> master
+        uses: ad-m/github-push-action@master
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          branch: master
+          force: true
+      - name: version
+        run: echo "::set-output name=version::$(python setup.py --version)"
```

actionlint

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)